src/views/blocks/rsvp/status/not-going.php

```diff
@@ -41,7 +41,7 @@
 		class="tribe-block__rsvp__status-button tribe-block__rsvp__status-button--not-going<?php if ( 'no' === $going ) { echo ' tribe-active'; }?>"
 		<?php echo disabled( 'no', $going, false ); ?>
 	>
-		<span><?php esc_html_e( 'Not going', 'event-tickets' ); ?></span>
+		<span><?php echo esc_html_x( 'Not going', 'Button label.', 'event-tickets' ); ?></span>
```

Comment:
I would suggest more context here, maybe something like "Label for the RSVP not going button"